openshift_cli_installer/cli.py

```diff
@@ -194,6 +194,15 @@
 """,
     type=click.Path(exists=True),
 )
+@click.option(
+    "--must-gather-output-dir",
+    help="""
+\b
+Path to must-gather output directory.
+must-gather will be collected when cluster installation fails.
+""",
+    type=click.Path(exists=True),
```

Comment:
Why `exists=True`? 
I think the user can pass `/tmp/my-must-gatther-dir` and we will create the dir.